Core/DISET/private/BaseClient.py

```diff
@@ -224,7 +243,18 @@ def _connect( self ):
       transport = gProtocolDict[ self.__URLTuple[0] ][ 'transport' ]( self.__URLTuple[1:3], **self.kwargs )
       retVal = transport.initAsClient()
       if not retVal[ 'OK' ]:
-        return S_ERROR( "Can't connect to %s: %s" % ( self.serviceURL, retVal ) )
```

Comment:
There is one server and it's down (or somehow you can't connect) you'll take 10 seconds before returning an error. Imagine the pain users will face when something like a CS server is down.